scripts/checkpatch_inc.sh

```diff
@@ -3,7 +3,8 @@
 CHECKPATCH="${CHECKPATCH:-checkpatch.pl}"
 # checkpatch.pl will ignore the following paths
 CHECKPATCH_IGNORE=$(echo core/lib/lib{fdt,tomcrypt} lib/lib{png,utils,zlib} \
-		core/arch/arm/plat-ti/api_monitor_index_a15.h)
+		core/arch/arm/plat-ti/api_monitor_index_a15.h \
+		core/arch/arm/plat-ti/api_monitor_index_a9.h)
```

Comment:
`core/arch/arm/plat-ti/api_monitor_index_a{9,15}.h` please, because it's more compact and we're already using the `{}` notation above.